keystore-service/src/main/java/com/stnetix/ariaddna/keystore/utils/KeyFactory.java

```diff
@@ -23,14 +24,16 @@ public static KeyFactory getKeyFactory() {
 
     private static final String KEYSTORE_PATH;
     private static final String DISABLED_KEYSTORE_PATH;
-    static final char[] PASS;
+    private static final char[] PASS;
     private static final AriaddnaLogger LOGGER;
+    private static final String KEYSTORE_FORMAT;
 
     static {
-        PASS = "password1".toCharArray();
+        PASS = UUID.randomUUID().toString().toCharArray();
```

Comment:
And each start server we have new keystore password :) I'am right?

We can make functionality for run only fist boot for generate RUID and persist this RUID to system DB.